### PR TITLE
Add num_bedroms to Tenancies controller

### PR DESCRIPTION
**Pull request by georgeschena**
Added num_bedroms to the tenancies controller

### CODE REVIEW

File: LBH.Data.Domain/Tenancy.cs [hunk 1]
```diff
@@ -108,6 +108,7 @@ public string PaymentRef
 
         public DateTime? StartDate { get; set; }
 
+        private string numberOfBedrooms;
```
Comment (kjdchapman) [inline LBH.Data.Domain/Tenancy.cs]: I'm not sure I understand this change. What is this private field doing?

Is this related to another change we made? I can't see anything in #117 that helps me either :confused:

Comment (georgeschena) [reply LBH.Data.Domain/Tenancy.cs]: We wanted to be consistent with how the other fields are declared within this file

Comment (georgeschena) [reply LBH.Data.Domain/Tenancy.cs]: The reason we weren't pulling in the number of bedrooms is because we forgot to add it into the controller

Comment (kjdchapman) [reply LBH.Data.Domain/Tenancy.cs]: Worried about us doing things for consistency without knowing what they're doing. 

My (limited) understanding of C# suggests this backing field isn't needed - do the tests pass without it?

**Pull request merged**